indy_node/test/auth_rule/test_auth_map.py

```diff
@@ -38,6 +38,14 @@ def test_auth_map_txn_author_agreement():
         assert rule_str in auth_map.auth_map.keys()
 
 
```

Comment:
Please add the following tests as well:

   - a unit test to indy_common/test/auth (test_txn_author_agreement_aml_with_new_auth_map.py)
   - an integration test for auth rule changing into indy_node/test/auth_rule/auth_framework

Reply:
ok